Ansible/roles/kvm/tasks/centos.yml

```diff
@@ -122,6 +123,26 @@
     - kvm
     - kvm-agent
 
+- name: Install centos-release
+  shell: /usr/bin/wget http://10.0.3.122/centos-release/{{ item }} -P /tmp
+  with_items:
+    - centos-release-7-9.2009.0.el7.centos.x86_64.rpm
+
+- name: Cleanup old repos
+  shell: rm /etc/yum.repos.d/CentOS- -f; rpm -ivh --replacepkgs --replacefiles /tmp/centos-release-7-9.2009.0.el7.centos.x86_64.rpm
+
+- name: ensure yum cache is cleared
+  shell: command="yum clean all"      
+
+- name: Ensure centos-release-qemu-ev is installed
+  yum: name=centos-release-qemu-ev state=present
+
+- name: Ensure qemu-kvm-ev is installed
+  yum: name=qemu-kvm-ev state=present
```

Comment:
I think in the main/sofia branch we shouldn't install qemu-kvm-ev by default; rest of the packages/changes are okay if they're general purpose (for ex. haveged etc). cc arch team @alexandremattioli @andrijapanicsb @NuxRo cc @davidjumani @nvazquez